examples/src/main/java/tachyon/examples/MasterConcurrencyPerformance.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.examples;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+import tachyon.TachyonURI;
+import tachyon.client.ClientContext;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.client.file.options.DeleteOptions;
+import tachyon.client.file.options.MkdirOptions;
+import tachyon.exception.TachyonException;
+import tachyon.util.FormatUtils;
+import tachyon.util.io.PathUtils;
+
+/**
+ * Contains performance test cases where a bunch of Tachyon clients concurrently
+ * operates on Tachyon master.
+ */
+public final class MasterConcurrencyPerformance {
+  private static enum OperationType {
+    CREATE_EMPTY_FILE,
+  }
+
+  private static interface OperationCallable {
```

Comment:
You can use `Function` in guava
http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Function.html